apps/outlookcalendar/outlook_calendar.star

```diff
@@ -198,7 +204,7 @@ def main(config):
             print(conflict_meeting_banner)
     else:
         conflict_meeting_banner = "No More Meetings for Today!"
-        display_calendar_date = time.parse_time(calendar_start_time).in_location("America/Detroit").format("Jan 2")
+        display_calendar_date = time.parse_time(calendar_start_time).in_location("America/Detroit").format("Jan 2")  # BUG HERE: Fix the Timezone (it's hardcoded)
```

Comment:
There's a "magic" config variable `$tz` when running in our backend. That holds the device's configured timezone.